dashboard/config/routes.rb

```diff
@@ -45,6 +45,7 @@ module OPS
 
   get 'maker/setup', to: 'maker#setup'
   get 'maker/discountcode', to: 'maker#discountcode'
+  post 'maker/apply', to: 'maker#apply'
```

Comment:
Open to feedback on the route name. One could argue that it should be something like `maker/discountcodeapply` (i.e. something with discountcode in the name).

Reply:
I think adding discount code in the name is probably a good idea, as I think it adds transparency as to what the application is for, but I also think it's fine either way.

Reply:
Seems fine to me.